airflow/providers/google/cloud/transfers/azure_fileshare_to_gcs.py

```diff
@@ -0,0 +1,217 @@
+        This parameter has been deprecated. You should pass the gcp_conn_id parameter instead.
+    :type dest_gcs_conn_id: str
+    :param dest_gcs: The destination Google Cloud Storage bucket and prefix
+        where you want to store the files. (templated)
+    :type dest_gcs: str
+    :param delegate_to: Google account to impersonate using domain-wide delegation of authority,
+        if any. For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :type delegate_to: str
+    :param replace: Whether you want to replace existing destination files
+        or not.
+    :type replace: bool
+    :param gzip: Option to compress file for upload
+    :type gzip: bool
+    :param google_impersonation_chain: Optional Google service account to impersonate using
+        short-term credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :type google_impersonation_chain: Union[str, Sequence[str]]
+
+
+    **Example**:
+
+    .. code-block:: python
+
+       azure_fileshare_to_gcs_op = AzureFileShareToGCSOperator(
+            task_id='azure_fileshare_to_gcs_example',
+            share='my-azure-share',
+            directory_name='mydir/anotherdir',
+            prefix='data/customers-201804',
+            dest_gcs_conn_id='google_cloud_default',
+            dest_gcs='gs://my.gcs.bucket/some/customers/',
+            replace=False,
+            gzip=True,
+            dag=my-dag
+       )
+
+    Note that ``share``, ``directory_name``, ``prefix``, ``delimiter`` and ``dest_gcs`` are
+    templated, so you can use variables in them if you wish.
+    """
+
+    template_fields: Iterable[str] = (
+        'share',
+        'directory_name',
+        'prefix',
+        'dest_gcs',
+        'google_impersonation_chain',
+    )
+
+    @apply_defaults
+    def __init__(
+        self,
+        *,
+        share_name,
+        directory_name=None,
+        prefix='',
+        wasb_conn_id='wasb_default',
+        gcp_conn_id='google_cloud_default',
+        dest_gcs_conn_id=None,
+        dest_gcs=None,
+        delegate_to=None,
+        replace=False,
+        gzip=False,
+        google_impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+
+        self.share_name = share_name
+        self.directory_name = directory_name
+        self.prefix = prefix
+        self.wasb_conn_id = wasb_conn_id
+
+        if dest_gcs_conn_id:
```

Comment:
I understand this is to keep consistency with existing operators but is there any point in adding deprecated arguments in brand new operators?

Reply:
No need for deprecation 👌

Reply:
ok, removed